### ISSUE TITLE
zig fmt: bad indentation for anonymous enum in parameter

### ISSUE THREAD

**Issue by andrewrk** [ziglang/zig#2125]
```zig
    fn reallocOrShrink(
        allocator: *Allocator,
        old_mem: []u8,
        old_align: u29,
        new_size: usize,
        new_align: u29,
        behavior: enum {
        shrink,
        realloc,
    },
    ) Error![]u8 {
```

**Comment by shritesh** [comment]
This too seems to be fixed by #2112

**Comment by andrewrk** [comment]
Confirmed
